server/controllers/middlewares/auth/index.ts

```diff
@@ -0,0 +1,15 @@
+import { RequestHandler } from 'express';
+import dotenv from 'dotenv';
+import { jwtVerify } from '../../../utils/jwt';
```

Comment:
Import this from the `utils` file
```suggestion
import { jwtVerify } from '../../../utils';
```